backend/Testing/SyncReverseProxy/SendReceiveServiceTests.cs

```diff
@@ -88,13 +91,28 @@ public void CloneBigProject()
     [InlineData(HgProtocol.Resumable, "manager")]
     public void CanCloneSendReceive(HgProtocol hgProtocol, string user)
     {
-        RunCloneSendReceive(hgProtocol, user, TestingEnvironmentVariables.ProjectCode);
+        RunCloneSendReceive(hgProtocol,
+            new SendReceiveAuth(user, TestingEnvironmentVariables.DefaultPassword),
```

Comment:
there's actually an admin auth field at the top, you can just reuse that here instead of making a new one

Reply:
So, I reduced the test to testing both protocols only with admin. That's approximately what you meant right?
Should I just remove the resumable version too? It's probably just a waste of time....

Reply:
oh I think I put this comment in the wrong spot. It should be for `CloneBigProject` as both parameters to the auth are the same as the field I referenced.

No I'd rather keep each around, it's just longer test time and it's important that it work for both use cases.

Reply:
Just both protocols? Or both users as well?

Reply:
protocols and users

Reply:
@hahn-kev OK, I think it's ready.